packages/starlight-blog/package.json

```diff
@@ -23,13 +23,13 @@
     "github-slugger": "2.0.0"
   },
   "devDependencies": {
-    "@astrojs/starlight": "0.11.0",
+    "@astrojs/starlight": "^0.13.0",
     "@playwright/test": "1.36.1",
-    "astro": "3.2.3"
+    "astro": "^4"
   },
   "peerDependencies": {
-    "@astrojs/starlight": ">=0.11.0",
-    "astro": ">=3.2.0"
+    "@astrojs/starlight": "0.11-0.13",
```

Comment:
I order to be sure that this is correct, one should in theory create a CI job matrix, testing all versions. But right now this seems overkill to me.